src/main/resources/assets/noindium/lang/cs_cz.json

```diff
@@ -0,0 +1,12 @@
+{
+  "header.noindium.indium": "Varování: Herní klient postrádá Indium",
+  "message.noindium.indium": "Varování: Jeden, nebo více z modifikací vaší hry by mohl mít problémy, způsobené použitím modu Sodium bez modu Indium. Tyto problémy jsou od malých vizuálních artefaktů, po havárii hry. Klikněte na jedno z těchto tlačítek, abyste stáhli Indium a vyhnuli se těmto i dalším závadám:",
+
+  "header.noindium.optifabric": "Varování: Optifabric přítomný v herním klientu",
+  "message.noindium.optifabric": "Varování: Jeden, nebo více z modifikací vaší hry by mohl mít problémy, způsobené použitím modu Optifabric. Tyto problémy jsou od malých vizuálních artefaktů, po havárii hry. Je doporučené vyhnout se těmto i dalším problémům. Pokud z nějakého důvodu potřebujete Optifine, podívejte se na list LambdAuroryných Optifine Alternativ:",
```

Comment:
Small nitpick, but is LambdAurora's nick correct here? It reads out as "LambdAurory" both to me and the translators I tested. Is this an error or a language quirk I'm unaware of?

Reply:
@Luligabi1 in our language we declense names. https://en.wikipedia.org/wiki/Czech_declension
I also used the second case of noun, we ask for the second case like this: "Whom? Which?". The list is LambAuroras so if all the rules go into play that is the result.

Reply:
When translating names we also look at similiar already czech name. I looked on "Jana" (because it ends on "a"). The second case of Jana is Janiných. So that is why I modified the ending -a to -yných (the first y is used because in czech after r we need to write y instead of i). To sum this up: My language has lot of rules and it is really hard. 😄

Reply:
Ahh, I see. Thanks for explaining it, I find it all very interesting :)